rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/SkeletalQrdaController.java

```diff
@@ -44,6 +43,8 @@
 	protected final ValidationService validationService;
 	protected final AuditService auditService;
 
+	protected abstract T respond(MultipartFile file, String checkedPurpose, HttpHeaders httpHeaders);
+
```

Comment:
If we do not have a new instance per request then these state variables could cause an issue.